tests/test_input.py

```diff
@@ -300,21 +302,21 @@ def mock_galaxy(disc_particles_all, halo_particles):
         vx_s=vel_s[:, 0] * (u.km / u.s),
         vy_s=vel_s[:, 1] * (u.km / u.s),
         vz_s=vel_s[:, 2] * (u.km / u.s),
-        m_s=mass_s * u.M_sun,
+        m_s=mass_s * 1e10 * u.M_sun,
```

Comment:
These masses are corrected in the mock_dm_halo and solid_disk, so it's no need for the unit's correction.